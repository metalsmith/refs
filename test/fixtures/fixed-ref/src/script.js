---
# a ref to the HTML page contents
refs:
  oneDeep: id:some-custom-id
---