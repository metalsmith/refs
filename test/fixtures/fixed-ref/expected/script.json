{
  "id": "script",
  "refs": {
    "oneDeep": {
      "id": "some-custom-id"
    }
  }
}